platforms/Windows/bundle/installer.wxs

```diff
@@ -0,0 +1,35 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal">
+  <Bundle
+    Name="!(loc.BundleName)"
+    Version="$(var.ProductVersion)"
+    Manufacturer="swift.org"
+    Compressed="$(IsBundleCompressed)"
+    UpgradeCode="710F1827-DA4A-4BF4-BDCE-D5F2D7C0DEF2">
```

Comment:
Is it no problem to have the same UpgradeCode for both architectures?

Would that prevent installing both the arm64 and amd64 binaries on an arm64 machine (which supports amd64 emulation)? (maybe not a real scenario, asking to understand)

Reply:
You're right; either architecture would upgrade its own and the other on an Arm64 machine. If making them side-by-side is a real scenario, then they need their own upgrade codes and some directory reorganization so they don't overlap. Windows Installer supports overlap for non-PE files like SDK headers but all the host binaries would need architecture-specific directories to avoid overlap. We can figure out the directories as part of the per-user work if you want it -- especially as we're already messing around with how the tree is organized by moving it out of Program Files.

Reply:
@compnerd Do you think we care about this?